pkg/kritis/crd/securitypolicy/violation.go

```diff
@@ -22,48 +22,54 @@ import (
 	"github.com/grafeas/kritis/pkg/kritis/apis/kritis/v1beta1"
 	"github.com/grafeas/kritis/pkg/kritis/constants"
 	"github.com/grafeas/kritis/pkg/kritis/metadata"
-)
-
-type Reason string
-
-// A list of security policy violations
-// TODO: Add Attestation checking violations
-const (
-	UnqualifiedImageViolation int = iota
-	FixUnavailableViolation
-	SeverityViolation
+	"github.com/grafeas/kritis/pkg/kritis/policy"
 )
 
 // Violation represents a vulnerability that violates an ISP
 type Violation struct {
 	Vulnerability metadata.Vulnerability
-	Violation     int
-	Reason        Reason
+	VType         policy.ViolationType
+	Msg           policy.Reason
```

Comment:
What's the background behind the terminology change here? If possible, I'd rather that the Policy & Violation structs use the same terminology.

Reply:
Violation.Violation sounded a little bit weird.
I had to change Reason -> Msg because you can't have a same name for method and variable.